### ISSUE TITLE
Make raylib even more directory independent

### ISSUE THREAD

**Issue by ratkingsminion** [raysan5/raylib#55]
The installer automatically installs raylib into C:\raylib - this is not desired by everyone. You can move the folder somewhere else of course, but the compiler scripts depend on this directory. It would be nice if those were relative.

**Comment by raysan5** [comment]
I know that not letting the user to choose the installation path could seem weird but after two years using raylib with my students, I took that decision because I faced the following problems doing otherwise:

* **PROBLEM 1**: Need to add raylib path to environment variable PATH: Some students had problems due to the security level configured in their systems (even adding a global var or a user var). Result: complaints: raylib doesn't work. Additionally, raylib installer touches system internals and it's something I try to avoid.
* **PROBLEM 2**: Using spaces in raylib path: Some students had problems when running the Notepad++ compiling scripts withing routes with spaces (additionally, this issue is platform dependant).
* **PROBLEM 3**: Notepad++ preconfigured compiling scripts depend directly on raylib installation directory. If we let the user choose the installation directory, we need to deal necessarily with PROBLEM 1 and PROBLEM 2 or make the user rewrite the path in every script or postprocess the scripts file after raylib installation. That's against my KISS principle.

After dealing with this issues for some time and taking into consideration that raylib is intended for amateur user (expert users can deal with it themselfs, moving the portable folder and reconfiguring paths themselfs), I took the decision to keep it as simple and constrained as possible. My student are very happy with it and noone complained about it; they only want to open it and see it work... and it does!

Nevertheless, I will thought about this issue and try to find an intermediate solution, at least letting the user to choose the installation directory (at its own risk).
